.github/workflows/release.yml

```diff
@@ -30,7 +30,9 @@ jobs:
                     - { tag: 'v3.0', php: '8.2', distro: bookworm, version-override: "", latest-tag: false }
                     - { tag: 'v3.1', php: '8.2', distro: bookworm, version-override: "", latest-tag: false }
                     - { tag: 'v3.2', php: '8.2', distro: bookworm, version-override: "", latest-tag: true }
+                    - { tag: 'v3.2', php: '8.3', distro: bookworm, version-override: "", latest-tag: true }
```

Comment:
Ps. I realize now, maybe this should be v3.3 so that v3.2 is still 8.2 + Imagick from official version?

Reply:
@youwe-petervanderwal agree, we should only use HEAD of master branch for 8.3 for now, the rest should stay the same. 
8.3 should be `3.x` only for now, we'll create a dedicated PR for release then.

Reply:
I've removed the `tag: 'v3.2', php: '8.3'` line